docs/usage/from-a-terminal.rst

```diff
@@ -609,8 +609,26 @@ This argument is suited to you!
 
     **Default value:** :code:`False`
 
-Want to get the full WHOIS record in your database?
-This argument is suited to you!
+The difference between :code:`False` or :code:`True` is whether
+we are saving a full dump of the `WHOIS` reply into the database.
+
+If you for some reason believes you need to fill up your database
+with a complete dump of the whois reply, this is the right value
```

Comment:
`s/reply/record` ?

Reply:
Yeah I did those to use the word reply to simplify it a bit as I actually had written record to begin with

The simplification should be understood as the transaction between what a human is doing from there own Linux terminal and into the test flow.

And until it is stored in the DB it is a reply...

But if you prefer `record` I'll of curse change it.

Reply:
All right, it's good to try to communicate with a human 😉